client/ayon_core/tools/publisher/publish_report_viewer/widgets.py

```diff
@@ -294,6 +294,70 @@ def _set_logs(self, logs):
         self._output_widget.setPlainText(text)
 
 
+class ClassInfoWidget(QtWidgets.QWidget):
+    def __init__(self, parent):
+        super(ClassInfoWidget, self).__init__(parent)
+
+        vertical_layout = QtWidgets.QVBoxLayout()
+        vertical_layout.setContentsMargins(8, 8, 8, 8)
+        self.setLayout(vertical_layout)
+
+        form_layout = QtWidgets.QFormLayout()
+        vertical_layout.addLayout(form_layout)
+
+        self._textedit_doc = QtWidgets.QTextEdit()
+        self._textedit_doc.setReadOnly(True)
+        self._textedit_doc.setMaximumHeight(300)
+        form_layout.addRow("Documentation", self._textedit_doc)
+
+        self._lineedit_file_path = QtWidgets.QLineEdit()
+        self._lineedit_file_path.setReadOnly(True)
+        form_layout.addRow("File Path", self._lineedit_file_path)
+
+        self._lineedit_plugin_type = QtWidgets.QLineEdit()
+        self._lineedit_plugin_type.setReadOnly(True)
+        form_layout.addRow("Plugin type", self._lineedit_plugin_type)
+
+        # self._lineedit_instances_labels = QtWidgets.QLineEdit()
+        # self._lineedit_instances_labels.setReadOnly(True)
+        # form_layout.addRow("Runs on instances", self._lineedit_instances_labels)
+
+        self._lineedit_families = QtWidgets.QLineEdit()
+        self._lineedit_families.setReadOnly(True)
+        form_layout.addRow("Families", self._lineedit_families)
+
+        vertical_layout.addStretch(100)
+
+        self._report_item = None
+        self._instance_filter = set()
+        self._plugin_filter = set()
+
+    def set_plugin_filter(self, plugin_filter):
+        try:
+            plugin_filter = list(plugin_filter)[0]
+        except Exception:
+            self._lineedit_file_path.setText(str())
+            self._lineedit_families.setText(str())
+            return
```

Comment:
This can be handled explicitly.
```suggestion
        if not plugin_ids:
            self._lineedit_file_path.setText("")
            self._lineedit_families.setText("")
            return

        plugin_id = list(plugin_ids)[0]
```